integration-tests/src/main/java/org/apache/polaris/service/it/env/PolarisApiEndpoints.java

```diff
@@ -28,7 +28,11 @@
  */
 public final class PolarisApiEndpoints implements Serializable {
 
-  public static String REALM_HEADER = "realm";
+  /**
+   * The header name for the realm ID. Tests must make sure that Polaris is configured with this
+   * header name.
+   */
+  public static String REALM_HEADER = "Polaris-Realm";
```

Comment:
Would you mind moving this to a (new) method on `PolarisServerManager` or `Server` so that it is possible to customize?